types/three/three-core.d.ts

```diff
@@ -6688,11 +6688,17 @@ export namespace SceneUtils {
     export function attach(child: Object3D, scene: Scene, parent: Object3D): void;
 }
 
+interface Vec2
+{
```

Comment:
Nitpicking: rest of file is egyptian braces.

Reply:
@FishOrBear could you please address the review comments?

Reply:
I think it's fine to merge but a style fix follow-up would be appreciated.

Reply:
Ok, Ill merge the PR since it has been open for some time. The style can be addressed on another PR.